test/test_operations.py

```diff
@@ -1552,6 +1553,55 @@ def test_clamp(self):
     self.assertEqual(b.data, xla_b.data.cpu())
 
 
+class XlaMNIST_Traced(nn.Module):
```

Comment:
The case choice (camel case + underscore) and the name of this are weird. Nothing about this is XLA specific, it's just instrumentation. Maybe call it `MNISTComparator`?

Reply:
SG